xep-0319.xml

```diff
@@ -25,6 +25,12 @@
   <shortname>idle</shortname>
   &tobias;
   <revision>
+    <version>1.0.1</version>
+    <date>2017-05-30</date>
+    <initials>egp</initials>
+    <remark><p>Add a precision about the &xep0082; profile used.</p></remark>
```

Comment:
Nit: "Be precise about the XEP-0082 profile used" sounds a bit better; precision is not plural/singular, so you can't have "a precision" (I'm not sure what that rule is actually called). I wouldn't add a link in the remarks either, although I'm not actually sure if it's against the DTD or not.